### PR TITLE
add component testing

### PR DESCRIPTION
**Pull request by CathrinTruchan**
Hello :)
I wrote some tests for three of my components. Happy about feedback. Thank you!

### CODE REVIEW

File: components/CreateFlowForm.test.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    const cancelButton = screen.getByRole("button", { name: /cancel/i });
+    await userEvent.click(cancelButton);
+    expect(cancelFunction).toHaveBeenCalledTimes(1);
+  });
+
+  it("adds a new text to text input", async () => {
+    render(<CreateFlowForm />);
+    const input = screen.getAllByRole("input");
+    await userEvent.type(input[0], "My new Flow");
+    expect(input[0]).toHaveValue("My new Flow");
```
Comment (Kheirah) [inline components/CreateFlowForm.test.js:26]:
```suggestion
    const input = screen.getByLabelText("name");
    await userEvent.type(input, "My new Flow");
    expect(input).toHaveValue("My new Flow");
```
A more precise way of getting hold of a specific input field would be to search for it by LabelText.

**Review by Kheirah — APPROVED** [review]
Very good!

Only one minor suggestion for getting specific input fields. Check the comment below.

File: components/CreateFlowForm.test.js (hunk 1 above)
Comment (CathrinTruchan) [reply components/CreateFlowForm.test.js:26]: Thanks for the tip, that looks much easier :)

**Comment by CathrinTruchan** [comment]
> Very good!
> 
> Only one minor suggestion for getting specific input fields. Check the comment below.

Thank you @Kheirah for your Feedback!

**Pull request merged**